### PR TITLE
Save seated section (app backend) (updated)

### LINKED ISSUE #68: [2pts] App Backend: Save Seated Section
**Fill in the blanks**:
As a developer, I would like to have a helper method that saves the section a user sits in, so that this method can be reused.

**Describe the solution you'd like:**
- [x] Parameter(s): Section int
- [x] Return: void

**Definition of Done - Check as you go!**
- [x] Tasks have been implemented
- [x] Documented
- [x] Passing Tests for each task
- [x] Pull Request made
- [x] Code Reviewed

### PR DESCRIPTION
**Pull request by MimiOlayeye**

### CODE REVIEW

File: app_backend/seat_test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+def test_set_seat():
+    section = []
+
+    for num in range(11): # testing 0 - 10
+        section.append(num)
+
+    for sections in section:
+        Tester = SaveSeating()
+        Test_success =  Tester.set_section_info(sections)
+        if (0< sections <=9): # Success Cases
```
Comment (RafaelPiloto10) [inline app_backend/seat_test.py]: nit: spacing

File: app_backend/seat_test.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+def test_set_seat():
+    section = []
+
+    for num in range(11): # testing 0 - 10
+        section.append(num)
+
+    for sections in section:
+        Tester = SaveSeating()
+        Test_success =  Tester.set_section_info(sections)
```
Comment (RafaelPiloto10) [inline app_backend/seat_test.py]: nit: variables should be named in lowercase

**Review by RafaelPiloto10 — CHANGES_REQUESTED** [review]
Some minor nits. Great job!

File: app_backend/sections.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+class Sections():
+    """
+
+    Stores the color labels associated with each numbered section within a dictionary
+    (sections range from 1 - 9) --> from @BrendaCano 's map of the DC
+
+    """
+    section_dict = {1: "red",
```
Comment (RafaelPiloto10) [inline app_backend/sections.py]: nit: consider rename to `color_dict` or something along that line

**Review by RafaelPiloto10 — CHANGES_REQUESTED** [review]

**Comment by RafaelPiloto10** [comment]
Remember to set the merge base when creating a pull request to `dev`

**Comment by MimiOlayeye** [comment]
My current branch is based on production :( Is there a simple way to switch it to dev? or do I just have to make a completely new branch @RafaelPiloto10

**Comment by RafaelPiloto10** [comment]
@MimiOlayeye you should be good -- the only files changed in this PR is the 3 you edited. I just meant that when you create a PR you have to specify what branch you want to merge your changes with, which should always be dev.

If you want to create a new branch, make sure you are always in `dev` to start and then create the new branch with `checkout -b <branch-name>`

**Review by RafaelPiloto10 — APPROVED** [review]

**Comment by MimiOlayeye** [comment]
Got it thanks! @RafaelPiloto10

**Review by DDVD233 — APPROVED** [review]
Looks good. Is this the fast API backend part? Looks like there's no fast API code.

**Comment by MimiOlayeye** [comment]
@DDavid233 no its app-backend

**Comment by RafaelPiloto10** [comment]
Good point @DDavid233, even though it is app-backend, we still need a way for the front end to alert the app-backend that we want it to save the seating function. @MimiOlayeye, do you think you could add a route for the app-backend to save the seated section? AKA implement your seated section code

**Review by Foxworth22 — APPROVED** [review]
What are we waiting on to merge this? @MimiOlayeye

**Pull request merged**